app/main.py

```diff
@@ -1,8 +1,44 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self, comfort_class: int, clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class: int = comfort_class
+        self.clean_mark: int = clean_mark
+        self.brand: str = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center: float = distance_from_city_center
+        self.clean_power: int = clean_power
+        self.average_rating: float = average_rating
+        self.count_of_ratings: int = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if self.clean_power > car.clean_mark:
```

Comment:
you are duplicating the code! method serve_cars has already done this check

Reply:
i have done this on purpose in case these methods will be used alone. without this check return value can be negative

Reply:
I can't agree with you. This is the same check and the method will not run if the condition is not passed. Please, give an example where the result can be negative

Reply:
for example:
```
audi = Car(comfort_class=4, clean_mark=9, brand='Audi')

wash_station = CarWashStation(
    distance_from_city_center=5,
    clean_power=6,  # clean_power is less then audi.clean_mark
    average_rating=3.5,
    count_of_ratings=6)

income = wash_station.calculate_washing_price(audi)

print(income)  # -8.4
```

Reply:
so we do not call this method to calculate income according to the condition of the problem) I think, in the future, we will be explained the difference between private, protected and ordinary methods in classes. you need to use the serve_cars method even if there is only one car